multiqc_ngi/multiqc_ngi.py

```diff
@@ -232,7 +232,11 @@ def get_ngi_project_metadata(self, pid):
 
         config.title = '{}: {}'.format(pid, p_summary['project_name'])
         config.project_name = p_summary['project_name']
-        config.output_fn_name = '{}_{}'.format(p_summary['project_name'], config.output_fn_name)
+        if config.analysis_dir and config.analysis_dir[0]=='qc_ngi':
+            infix = 'qc'
+        else:
+            infix = 'pipeline'
+        config.output_fn_name = f'{p_summary['project_name']}_{infix}_{config.output_fn_name}'
```

Comment:
Doesn't the single quotes around project_name break the f-string?

Reply:
Absolutely. I've changed it 😅